src/Symfony/Bundle/FrameworkBundle/Tests/Translation/TranslatorTest.php

```diff
@@ -296,14 +296,3 @@ private function createTranslator($loader, $options, $translatorClass = '\Symfon
         );
     }
 }
-
-class TranslatorWithInvalidLocale extends Translator
-{
-    /**
-     * {@inheritdoc}
-     */
-    public function setLocale($locale)
-    {
-        $this->locale = $locale;
```

Comment:
why remove this tests? a custom translator can still return an invalid locale if it overwrites `getLocale` instead (or getLocale and setLocale).